client/state/selectors/get-atomic-transfer.js

```diff
@@ -6,5 +6,5 @@
 import { get } from 'lodash';
 
 export default ( state, siteId ) => {
-	return get( state, [ 'atomicTransfer', siteId ], {} );
+	return get( state, [ 'atomicTransfer', siteId, 'atomicTransfer' ], {} );
```

Comment:
@jhnstn I had to add this to the selector to get the actual transfer object. Just a heads up in case `getAtomicTransfer()` was intended to also return the `fetchingTransfer` state.

Reply:
Thanks, yeah I don't think we need the fetching state on the transfer object
👍